modules/services/window-managers/hyprland.nix

```
@@ -103,6 +103,11 @@ in {
         ];
         description = "Extra commands to be run after D-Bus activation.";
       };
+
+      xdgAutostart = lib.mkEnableOption ''
+        autostart of applications using
+        {manpage}`systemd-xdg-autostart-generator(8)`
+      '';
```

```suggestion
        {manpage}`systemd-xdg-autostart-generator(8)`'';
```

It renders with an extra space if you don't end the string on the last line.